robot_ws/src/cloudwatch_robot/launch/await_commands.launch.py

```diff
@@ -0,0 +1,24 @@
+import os
+import sys
+
+import launch
+import launch_ros.actions
+from ament_index_python.packages import get_package_share_directory
+
+def generate_launch_description():
+    launch_actions = [
+        launch.actions.DeclareLaunchArgument(
+            name='use_sim_time',
+            default_value='true'
+        ),
+        launch.actions.IncludeLaunchDescription(
+            launch.launch_description_sources.PythonLaunchDescriptionSource(
+                os.path.join(get_package_share_directory('cloudwatch_robot'), 'launch/monitoring.launch.py')
```

Comment:
Use os.path.join to build the entire path:
`os.path.join(get_package_share_directory('cloudwatch_robot'), 'launch', 'monitoring.launch.py')`